src/common/sidebar/components/Logs.tsx

```diff
@@ -1,7 +1,15 @@
-import { Box } from "@chakra-ui/layout";
+import { Box, Heading, HStack } from "@chakra-ui/layout";
 
-function Logs() {
-  return <Box h="100%" w="100%" bg="green" />;
+interface LogProps{
+  logs: string []
 }
 
-export default Logs;
+function Logs({ logs }: LogProps) {
+  return <Box border="1px" borderColor="black" boxShadow="dark-lg" h="95%" w="95%" bg="white">{logs.map((log) => {
```

Comment:
Instead of using height and width 95% use padding `Xem`, where X is a number (`1em` or whatever looks best.)